assembl/static2/css/errorPages.css

```diff
@@ -0,0 +1,217 @@
+	font-family: monospace, monospace;
+	font-size: 1em
+}
+dfn {
+	font-style: italic
+}
+mark {
+	background-color: #ff0;
+	color: #000
+}
+small {
+	font-size: 80%
+}
+sub, sup {
+	font-size: 75%;
+	line-height: 0;
+	position: relative;
+	vertical-align: baseline
+}
+sub {
+	bottom: -.25em
+}
+sup {
+	top: -.5em
+}
+audio, video {
+	display: inline-block
+}
+audio:not([controls]) {
+	display: none;
+	height: 0
+}
+img {
+	border-style: none
+}
+svg:not(:root) {
+	overflow: hidden
+}
+button, input, optgroup, select, textarea {
+	font-family: sans-serif;
+	font-size: 100%;
+	line-height: 1.15;
+	margin: 0
+}
+button, input {
+	overflow: visible
+}
+button, select {
+	text-transform: none
+}
+[type=reset], [type=submit], button, html [type=button] {
+	-webkit-appearance: button
+}
+[type=button]::-moz-focus-inner, [type=reset]::-moz-focus-inner, [type=submit]::-moz-focus-inner, button::-moz-focus-inner {
+	border-style: none;
+	padding: 0
+}
+[type=button]:-moz-focusring, [type=reset]:-moz-focusring, [type=submit]:-moz-focusring, button:-moz-focusring {
+	outline: 1px dotted ButtonText
+}
+fieldset {
+	border: 1px solid silver;
+	margin: 0 2px;
+	padding: .35em .625em .75em
+}
+legend {
+	box-sizing: border-box;
+	color: inherit;
+	display: table;
+	max-width: 100%;
+	padding: 0;
+	white-space: normal
+}
+progress {
+	display: inline-block;
+	vertical-align: baseline
+}
+textarea {
+	overflow: auto
+}
+[type=checkbox], [type=radio] {
+	box-sizing: border-box;
+	padding: 0
+}
+[type=number]::-webkit-inner-spin-button, [type=number]::-webkit-outer-spin-button {
+	height: auto
+}
+[type=search] {
+	-webkit-appearance: textfield;
+	outline-offset: -2px
+}
+[type=search]::-webkit-search-cancel-button, [type=search]::-webkit-search-decoration {
+	-webkit-appearance: none
+}
+::-webkit-file-upload-button {
+	-webkit-appearance: button;
+	font: inherit
+}
+details, menu {
+	display: block
+}
+summary {
+	display: list-item
+}
+canvas {
+	display: inline-block
+}
+template {
+	display: none
+}
+[hidden] {
+	display: none
+}
+body, html {
+	width: 100%;
+	height: 100%;
+	background-color: #21232a
+}
+body {
+	color: #fff;
+	text-align: center;
+	text-shadow: 0 2px 4px rgba(0, 0, 0, .5);
+	padding: 0;
+	min-height: 100%;
+	-webkit-box-shadow: inset 0 0 75pt rgba(0, 0, 0, .8);
+	box-shadow: inset 0 0 75pt rgba(0, 0, 0, .8);
+	display: table;
+	font-family: "Open Sans", Arial, sans-serif
+}
+h1 {
+	font-family: inherit;
+	font-weight: 500;
+	line-height: 1.1;
+	color: inherit;
+	font-size: 36px
+}
+h1 small {
+	font-size: 68%;
+	font-weight: 400;
+	line-height: 1;
+	color: #777
+}
+a {
+	text-decoration: none;
+	color: #fff;
+	font-size: inherit;
+	border-bottom: dotted 1px #707070
+}
+.lead {
+	color: silver;
+	font-size: 21px;
+	line-height: 1.4
+}
+.cover {
+	display: table-cell;
+	vertical-align: middle;
+	padding: 0 20px
+}
+footer {
+	position: fixed;
+	width: 100%;
+	height: 40px;
+	left: 0;
+	bottom: 0;
+	color: #a0a0a0;
+	font-size: 14px
+}
```

Comment:
This file is completely outside of the V2 and V1 world. It is used for all jinja2 templates that will replace when we get 404s, 500s, etc errors from Pyramid.